apps/condo/package.json

```diff
@@ -79,6 +79,7 @@
     "@tinymce/tinymce-react": "^4.1.0",
     "@types/lodash": "4.14.186",
     "@types/wavesurfer.js": "^6.0.6",
+    "@vicimpa/rubles": "^0.1.18",
```

Comment:
adding another library for one locale specific task seems not a good idea. Why not simply inline the code? It was 100 lines or smth

Why inlining is better
- less attack surface for supply chain attack (see XZ case)
- less bloated node_modules
- less stuff to update